README.md

```diff
@@ -83,8 +83,8 @@ Symbol Definitions:
 
 - [x] 2024Q1 - Enable producing the [zkEVM](https://github.com/kroma-network/zkevm-circuits) proof.
 - [ ] 2024Q2 - Replace Halo2 with Tachyon in [Kroma](https://kroma.network/) mainnet.
-- [ ] 2024Q3 - Implement Halo2 GPU.
-- [ ] 2024Q4 - Implement Halo2 Folding Scheme.
+- [ ] 2024Q3a - Implement and optimize Plonky3 in Tachyon.
+- [ ] 2024Q3b - Replace zkEVM with [SP1](https://github.com/succinctlabs/sp1) zkVM in [Kroma](https://kroma.network/) mainnet.
```

Comment:
For convenience, how about marking it for Q2 and merging this PR after this Friday?